webapp/pages/moderation/index.vue

```diff
@@ -82,6 +82,14 @@
           {{ scope.row.submitter.name }}
         </nuxt-link>
       </template>
+      <!-- createdAt -->
+      <template slot="createdAt" slot-scope="scope">
```

Comment:
Seems to be a deprecated API: https://vuejs.org/v2/guide/components-slots.html#Scoped-Slots-with-the-slot-scope-Attribute